<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Distance3DView</class>
 <widget class="QWidget" name="Distance3DView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance View</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="PARAM_GRP">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="V_SPACER_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="PARAM_LAYOUT">
        <property name="labelAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <item row="1" column="0" colspan="2">
         <spacer name="V_SPACER_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="HORIZONTAL_LBL">
          <property name="text">
           <string>Horizontal</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="H_ANGLE_SPIN">
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="DirectionType" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <spacer name="V_SPACER_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="VERTICAL_LBL">
          <property name="text">
           <string>Vertical</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="V_ANGLE_SPIN">
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="DirectionType" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <spacer name="V_SPACER_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="SCALE_LBL">
          <property name="text">
           <string>Scale</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="SCALE_SLIDER">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <spacer name="V_SPACER_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="PROJECTION_LBL">
          <property name="text">
           <string>Projection</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="PROJECTION_LST"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="DOWNSAMPLE_LBL">
          <property name="text">
           <string>Downsample</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="DOWNSAMPLE_LST"/>
        </item>
        <item row="9" column="0" colspan="2">
         <spacer name="V_SPACER_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="CONSECUTIVE_CHK">
          <property name="text">
           <string>Consecutive</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="REFRESH_BTN">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="V_SPACER_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="DISTANCE_DISPLAY" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
